src/library/SubmitTx/index.tsx

```diff
@@ -75,52 +72,32 @@ export const SubmitTx = ({
   }, []);
 
   return (
-    <Wrapper noMargin={noMargin}>
-      <div className="inner">
-        <p className="sign">
-          <span className="badge">
-            <FontAwesomeIcon icon={faPenToSquare} className="icon" />
-            {signingOpts.label}
-          </span>
-
-          {signingOpts.who?.name}
-
-          {notEnoughFunds && (
-            <span className="notEnough">
-              / &nbsp;
-              <FontAwesomeIcon
-                icon={faWarning}
-                className="danger"
-                transform="shrink-1"
-              />{' '}
-              <span className="danger">
-                {t('notEnough', { ns: 'library' })} {unit}
-              </span>
-            </span>
-          )}
-        </p>
-
-        <section className="foot">
-          {requiresManualSign(sender) ? (
-            <ManualSign
-              uid={uid}
-              onSubmit={onSubmit}
-              submitting={submitting}
-              valid={valid}
-              submitText={submitText}
-              buttons={buttons}
-            />
-          ) : (
-            <Default
-              onSubmit={onSubmit}
-              submitting={submitting}
-              valid={valid}
-              submitText={submitText}
-              buttons={buttons}
-            />
-          )}
-        </section>
-      </div>
-    </Wrapper>
+    <Tx
+      margin
+      label={signingOpts.label}
+      name={signingOpts.who?.name || ''}
+      notEnoughFunds={notEnoughFunds}
+      dangerMessage={`${t('notEnough', { ns: 'library' })} ${unit}`}
+      requiresManualSign={requiresManualSign(sender)}
+      ManualSign={
+        <ManualSign
+          uid={uid}
+          onSubmit={onSubmit}
+          submitting={submitting}
+          valid={valid}
+          submitText={submitText}
+          buttons={buttons}
+        />
+      }
+      DefaultSign={
```

Comment:
Why are we passing both `ManualSign` and `DefaultSign` here? Only one of them will ever be active. Please can you amend the component on the UI side to only take the signer we want, and then come back to this PR to amend his.